templates/nutrition_facts_table.tt.html

```diff
@@ -0,0 +1,79 @@
+[% IF comparisons.defined %]
+    <p> [% lang('nutrition_data_comparison_with_categories') %] </p>
+
+        [% FOREACH comparison IN comparisons %]
+            <label style="display:inline;font-size:1rem;">
+				<input type="checkbox" id="[% comparison.col_id %]" name="[% comparison.col_id %]" value="on" [% IF comparison.checked %]checked="checked"[% END %] class="show_comparison"> [% comparison.name %]</label>
```

Comment:
```suggestion
                <input type="checkbox" id="[% comparison.col_id %]" name="[% comparison.col_id %]" value="on" [% IF comparison.checked %]checked="checked"[% END %] class="show_comparison"> 
                [% comparison.name %]
            </label>
```

Reply:
That's it. Do I have to clic on "Commit suggestion"?

Reply:
@VaiTon  @CharlesNepote what's the recommended way? Making all these suggested changes in a single commit OR directly clicking 'Commit suggestion' for every suggested change?